apps/frontend/src/components/Decorators/Draggable.tsx

```diff
@@ -1,15 +1,13 @@
-import { useMitt } from '@/plugins/mitt/react-mitt';
-
 interface DraggableProps {
     children: React.ReactElement;
     type: string;
 }
 
 const Draggable: React.FunctionComponent<DraggableProps> = (props) => {
-    const emitter = useMitt();
+    const emitter = window.getEmitter();
```

Comment:
This MUST NOT be done.
Otherwise what's the utility of the `MittProvider` ?
The `emitter`'s getter in the `window` element should only be accessed inside `Preview` instances and subcomponents.